config/class-site-details-index.php

```diff
@@ -349,8 +349,21 @@ public function get_site_details() {
 	 */
 	public function put_site_details() {
 		$site_details = $this->get_site_details();
+		$url          = null;
+		$args         = null;
 
-		if ( defined( 'SERVICES_API_URL' ) && defined( 'SERVICES_AUTH_TOKEN' ) && ! empty( SERVICES_AUTH_TOKEN ) ) {
+		if ( defined( 'SDS_API_URL' ) && defined( 'SDS_AUTH_TOKEN' ) && ! empty( SDS_AUTH_TOKEN ) ) {
+			$url = rtrim( SDS_API_URL, '/' ) . '/sites';
+
+			$args = array(
+				'method'  => 'PUT',
+				'body'    => wp_json_encode( $site_details ),
+				'headers' => array(
+					'Authorization' => 'Bearer ' . SDS_AUTH_TOKEN,
+					'Content-Type'  => 'application/json',
+				),
+			);
```

Comment:
The whole args array seems to be identical to the else branch except the token. What do you think of setting something like:
```
$auth_token = SDS_AUTH_TOKEN
```

Then constructing the args array only before `vip_safe_wp_remote_request`, so in shared code for both conditions.